src/VisualStudio/Core/Impl/CodeModel/FileCodeModel.cs

```diff
@@ -40,7 +40,7 @@ public sealed partial class FileCodeModel : AbstractCodeModelObject, EnvDTE.File
             ITextManagerAdapter textManagerAdapter)
         {
             // Keep track that code model was accessed.  We want to get a sense of how widespread usage of it still is.
-            Logger.Log(FunctionId.CodeModel_FileCodeModel_Create);
+            Logger.Log(FunctionId.CodeModel_FileCodeModel_Create, logLevel: RoslynLog.LogLevel.Information);
```

Comment:
what is this for?

Reply:
I cannot tell you the history, but only loglevel of information or higher gets logged to telemetry, see https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Core/Def/Telemetry/Shared/TelemetryLogger.cs#L166